test/test_runner.rb

```diff
@@ -7,7 +7,27 @@
 
 # Unit test class for PmdTester::Runner
 class TestRunner < Test::Unit::TestCase
+  def setup
+    `rake clean`
+  end
+
   include PmdTester
+  def test_option_miss(argv, expects)
```

Comment:
I'd call this method something like `run_and_assert_error_messages`. I first mistakenly thought, this would be a test of its own...